views/scss/qtiCreator/_grid-editor.scss

```diff
@@ -225,6 +225,13 @@
     position: relative;
     margin-top: 18px;
     padding: 3px;
+    @extend %clearfix;
+    &:before {
```

Comment:
https://github.com/oat-sa/tao-core-ui-fe/blob/ab598265609cd90ee148c7bf2e2a0732e13e2ad3/scss/inc/_variables.scss#L13-L21
before and after already included in extend `%clearfix`